.github/workflows/spec-parser.yml

```diff
@@ -68,28 +59,28 @@ jobs:
       - name: generate docs with Ontospy
         run : |
           cd Ontospy
-          mkdir -p /tmp/auto-generated
```

Comment:
I think we should keep this for now until we get the spdx-spec repo up and running